src/containerapp/azext_containerapp/tests/latest/utils.py

```diff
@@ -37,21 +37,15 @@ def prepare_containerapp_env_for_app_e2e_tests(test_cls, location=TEST_LOCATION)
     return managed_env["id"]
 
 
-def create_containerapp_env(test_cls, env_name, resource_group, location=None, subnetId=None):
```

Comment:
Please don't change this function if not required.
PR https://github.com/Azure/azure-cli-extensions/pull/7155 will use subnetId.

Reply:
Whoops - yes I meant to undo that change thanks :)